css/style.css

```diff
@@ -1,55 +1,58 @@
-html {
-    font-size: 120%;
-    margin: 60px;
-  }
+body {
+  font-size: 120%;
+  margin: 60px;
+  margin: 0;
+  padding: 0;
+  text-align: center;
+  font-family: cursive;
+}
+
+footer {
+  text-align: left;
+  background-color: pink;
+  border: 3px solid purple;
+  border-radius: 5px;
+}
+
+/* fonts */
+
+h1 {
+  color: mediumorchid;
+  font-family: cursive;
+  font-size: 60px;
+}
+
+h2 { 
+  color: mediumorchid;
+  font-family: cursive;
+}
+
+section {
+  max-width: 780px;
+  margin: 0 auto;
+}
+
+/* class selectors */
+
+.main-text {
+  text-align: justified;
+}
+
+.nobullet {
+  list-style: none;
+}
+/* id selectors */
+
+#main-logo {
+  border: 3px solid black;
+  border-radius: 7px;
   
-  body {
-   
-    margin: 0;
-    padding: 0;
-    text-align: center;
-    font-family: cursive;
-  }
-  footer {
-    text-align: left;
-    background-color: pink;
-    border: 3px solid purple;
-    border-radius: 5px;
-  }
+}
+#katie-image {
+  border: 2px solid green;
+  Border-radius: 100px;
   
-  h1 {
-    color: mediumorchid;
-    font-family: cursive;
-    font-size: 60px;
-  }
-  h2 { 
-    color: mediumorchid;
-    font-family: cursive;
-  }
-  
-  /* class selectors */
-  
-  .main-text {
-    text-align: justified;
-    margin-left: 340px;
-    margin-right: 340px;
```

Comment:
Using large fixed margins means that if you resize the screen, your text can end up 1 character wide!